samza-log4j2/src/test/java/org/apache/samza/logging/log4j2/TestStreamAppender.java

```diff
@@ -68,13 +68,12 @@ public void testDefaultSerde() {
   @Test
   public void testNonDefaultSerde() {
     System.setProperty("samza.container.name", "samza-container-1");
-    String streamName = StreamAppender.getStreamName("log4jTest", "1");
     Map<String, String> map = new HashMap<String, String>();
     map.put("job.name", "log4jTest");
     map.put("job.id", "1");
     map.put("serializers.registry.log4j-string.class", LoggingEventStringSerdeFactory.class.getCanonicalName());
     map.put("systems.mock.samza.factory", MockSystemFactory.class.getCanonicalName());
-    map.put("systems.mock.streams." + streamName + ".samza.msg.serde", "log4j-string");
+    map.put("systems.mock.streams.__samza_log4jTest_1_logs.samza.msg.serde", "log4j-string");
```

Comment:
just curious why this is being hardcoded..

Reply:
Because `getStreamName` function is changed from `protected static ` to `protected` as static method cannot be overridden.
BTW, this static use case only exists here in samza's code. Seems good to remove static keyword.